WcaOnRails/app/views/static_pages/about.html.erb

```diff
@@ -15,14 +15,15 @@
   <p><%= t('about.structure.board_html') %></p>
 
   <ul>
-    <% @board_members.order(:name).each do |board_member| %>
-      <li><%= board_member.name %> - <%= board_member.region %></li>
+    <% Team.find_by_friendly_id(Team::BOARD_FRIENDLY_ID).current_members.includes(:user).map(&:user).sort_by(&:name).each do |board_member| %>
+      <li><%= board_member.name %> - <%= board_member.country.name %></li>
     <% end %>
   </ul>
 
   <p><%= t('about.structure.committees') %>
   <ul>
     <% Team.order(:rank).each do |team| %>
+      <% next if team.friendly_id == 'board' %>
```

Comment:
@jfly Hopefully this is what you meant :)

Reply:
Looking better! In light of my other suggestion, what about changing this to `next if team == Team.board`?